apps/golden-sample-app/src/app/app.component.ts

```diff
@@ -14,9 +15,21 @@ export class AppComponent {
 
   constructor(
     private oAuthService: OAuthService,
-    public layoutService: LayoutService
+    public layoutService: LayoutService,
+    private router: Router
   ) {
     this.isAuthenticated = oAuthService.hasValidAccessToken();
+    // Listening for navigation events
+    router.events.forEach((event) => {
+      if(event instanceof NavigationEnd) {
+        (window as any).trackingEventsFake('change language:', event);
```

Comment:
Why language? Do you mean `location`?

Reply:
You are right, it was just a copy-paste issue.

Reply:
Fixed.